extensions/vscode/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.2.4
+
+- Housekeeping and maintenance updates.
```

Comment:
The previous changelog had three changes that do impact the product:

- chore(deps): bump semver from 7.5.4 to 7.6.0 in /extensions/vscode ([#1286](https://github.com/VeryGoodOpenSource/dart_frog/pull/1286))
- chore(deps): bump semver from 7.6.0 to 7.6.2 in /extensions/vscode ([#1367](https://github.com/VeryGoodOpenSource/dart_frog/pull/1367))
- chore(deps): bump semver from 7.6.2 to 7.6.3 in /extensions/vscode ([#1459](https://github.com/VeryGoodOpenSource/dart_frog/pull/1459))

Maybe we can combine them into:
- chore(deps): bump semver from 7.6.2 to 7.6.3 in /extensions/vscode ([#1286](https://github.com/VeryGoodOpenSource/dart_frog/pull/1286), [#1367](https://github.com/VeryGoodOpenSource/dart_frog/pull/1367), [#1459](https://github.com/VeryGoodOpenSource/dart_frog/pull/1459))

Or simply just include the change that superseded all of the previous ones:
- chore(deps): bump semver from 7.6.2 to 7.6.3 in /extensions/vscode ([#1459](https://github.com/VeryGoodOpenSource/dart_frog/pull/1459))

In addition I would also keep the semantic commits in the CHANGELOG for consistency and in case

```suggestion
- chore(deps): bump semver from 7.6.2 to 7.6.3 in /extensions/vscode ([#1459](https://github.com/VeryGoodOpenSource/dart_frog/pull/1459))
- chore(deps-dev): housekeeping and maintenance updates
```